app/components/login-form.hbs

```diff
@@ -1,25 +1,17 @@
-<form class="formclass">
+<form class="formclass" {{on "submit" this.signIn}}>
   <h3 class="sign-in">Sign In</h3>
   <div class="email-id mb-3">
-    <input
-      id="email"
-      type="email"
-      class="form-control"
-      placeholder="Enter Email"
-      aria-describedby="emailHelp"
-    />
-    <div class="error1"><small class="error">Invalid Email ID </small></div>
+    <Input type="email" class="form-control" id="email-field" placeholder="Enter Email" @value={{this.email}} />
+    <small class="text-danger email-error">{{this.emailError}}</small>
   </div>
   <div class="pwd mb-3">
-    <input
-      type="password"
-      class="form-control"
-      id="password"
-      placeholder="Enter Password"
-    />
-    <div class="error1"><small class="error">Invalid Password</small></div>
+    <Input type="password" class="form-control" id="password-field" placeholder="Enter Password"
+      @value={{this.password}} />
+    <small class="text-danger password-error">{{this.passwordError}}</small>
 
   </div>
-  <LinkTo @route="dashboard" class="btn btn-primary">Sign In</LinkTo>
-  <div id="emailHelp" class="form-text"><p>Forgot password</p></div>
+  <button type="submit" class="btn btn-primary" {{on "click" this.signIn}}>Sign In</button>
+  {{!-- <div id="emailHelp" class="form-text">
+    <p>Forgot password</p>
```

Comment:
check on this commented lines...if it is unwanted, remove

Reply:
removed it

Reply:
Aren't we having the forgot password link then?

Reply:
we need to have rahul told that he will do it later